webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactPasswordStrength from 'react-password-strength';
+
+import CommonForm from '../common/forms/CommonForm';
+
+import './PasswordStrength.scss';
+
+export default class PasswordStrength extends React.Component {
+  render() {
+    const {
+      updatePassword,
+      updatePasswordConfirmation,
+      doesPasswordsMatch,
+      data: {
+        className, id, name, verify, error, userInputIds,
+      },
+    } = this.props;
+
+    const userInputs =
+      userInputIds && userInputIds.length > 0
+        ? userInputIds.map(input => document.getElementById(input).value)
+        : [];
```

Comment:
Just out of curiosity: What's shown in the strength meter when the password contains the username? I tried to simulate that but without any luck.

Reply:
It should show 'WEAK'. I just tested it and it works.

Reply:
I see, you're right. I confirm it works.